README.md

```diff
@@ -16,12 +16,12 @@ It is **serializable** to JSON and **deserializable** from JSON, it's immutable
 // Just use @AutoParcel annotations if you need AutoParcel
 
 @AutoValue
-@JsonDeserialize(builder = AutoValue_Tweet.Builder.class)
+@JsonDeserialize(builder = Tweet.Builder.class)
```

Comment:
Wait, `@JsonSerialize(as = Tweet.class)`

Reply:
Sure, it should have both serializer and deserializer.